shipyard_blueprints/trello/shipyard_trello/__init__.py

```diff
@@ -1 +1 @@
-from .trello import TrelloClient
\ No newline at end of file
+from .trello import TrelloClient, TicketNotFound
```

Comment:
fixed the folder structure. Some how I must've moved the file to an upper folder.
it used to be 
-trello
--poetry lock
--readme
--toml
-- trello.py
--init
--shipyard_trello
---cli


i've moved the init file and trello.py inside the shipyard_trello directory